crates/lib/src/status.rs

```diff
@@ -331,6 +332,33 @@ pub(crate) fn get_status(
 /// Implementation of the `bootc status` CLI command.
 #[context("Status")]
 pub(crate) async fn status(opts: super::cli::StatusOpts) -> Result<()> {
+    // Log the status query operation to systemd journal
```

Comment:
I don't think we should log status operations, that feels like it'll be noisy - who would ever want to know that a status was requested?

It's a bit like Unix atime - having a read-only operation perform a write is weird.

Reply:
Made changes to make logs more focused so we get actionable information about system state changes without journal clutter from routine queries.